src/Kunstmaan/NodeSearchBundle/Helper/ElasticSearchUtil.php

```diff
@@ -12,6 +14,16 @@
      */
     public static function useVersion6()
     {
-        return (PHP_MAJOR_VERSION == 7 && !class_exists('\Elastica\Tool\CrossIndex'));
+        $client = ClientBuilder::create()->build();
+        $info = $client->info();
```

Comment:
We can cache this in a static variable. Otherwise it's gets called multiple times (e.g. with cache:clear).

`private static $esClientInfo;`

  ```
  $info = self::getESVersionInfo();

/**
     * @return array
     */
    private static function getESVersionInfo()
    {
        if (null === self::$esClientInfo) {
            $client = ClientBuilder::create()->build();
            $info = $client->info();
            self::$esClientInfo = $info;
        } else {
            $info = self::$esClientInfo;
        }

        return $info;
    }
```